packages/tailwind/helpers/helpers.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @param value any value for converting it to raw string
+ * @param nestingLevel current nesting level (is needed for correct indents)
+ * @returns raw string value
+ */
+export const convertValueToString = (value: any, nestingLevel = 1): string => {
```

Comment:
Just curious, but why not `JSON.stringify`? :)

Reply:
I've tried it first of all, but it makes not exactly what we need (or we will be forced to ask our users to make JSON.parse and that's weird).